README.md

```diff
@@ -19,20 +19,34 @@ Create Grafana plugins with ease.
 If something doesn’t work, please [file an issue](https://github.com/grafana/create-plugin/issues/new).<br />
 If you have questions or need help, please ask in [GitHub Discussions](https://github.com/grafana/create-plugin/discussions).
 
-## Create a new plugin
+## Create a new grafana plugin
 
-### Quick overview
+### Scaffolding
 
-The following commands scaffold a Grafana plugin inside the `./my-plugin` folder:
+* `mkdir -p $HOME/grafana-plugins && cd $HOME/grafana-plugins`
+* Run `yarn create @grafana/plugin` and follow the questions
+* Cd to your new plugin directory. e.g.: `cd my-org-my-plugin-name`
+* Install the dependencies: `yarn install`
+* Start developing: `yarn dev` (will watch changes in your plugin code and rebuild)
 
-```bash
-mkdir my-plugin && cd my-plugin
-yarn create @grafana/plugin
-yarn install
-yarn dev
-```
+### Run your plugin inside grafana (docker example)
+
+* Run grafana inside docker:
+
+  ```bash
+  docker run -p 3000:3000 -v "$HOME"/grafana-plugins:/var/lib/grafana/plugins --name=grafana grafana/grafana:9.1.2
```

Comment:
Doesn't this package create a docker-compose for development purposes?

Reply:
Yeah, probably they should be just able to run `docker-compose up` ([docker-compose.yaml](https://github.com/grafana/create-plugin/blob/b198dc218f6d2810b93fda0c468c4005dbcf43ae/templates/common/docker-compose.yaml#L8)) instead of a docker run command? I think it also refreshes the browser on every file change ([Dockerfile](https://github.com/grafana/create-plugin/blob/882ee268814bf045d601d32a9913fa0c2c8d2b86/templates/common/.config/Dockerfile#L14-L15))?